src/App.tsx

```diff
@@ -1,9 +1,12 @@
-import "./App.css";
+import { useState } from "react";
+import SplashScreen from "./components/SplashScreen";
+import "./style/App.css";
 
 function App() {
+  const [order, setOrder] = useState({});
   return (
     <div>
-      <h1>Panini Creator</h1>
+      <SplashScreen order={Object.keys(order).length > 0 ? true : false}></SplashScreen>
```

Comment:
**nitpick:** This is not too descriptive. As a developer who reads it, it's very hard to predict that this prop does. I suggest you to differ it by more meaningful prop as for eg. `isPaniniOrdered` as boolean.